src/styles/colors.ts

```diff
@@ -0,0 +1,5 @@
+export const Colors = {
+  rosado: '#FF197B',
+  backGround: '#F0F0F0',
+  azul: '#1F86FF',
```

Comment:
Please keep the consistency with the code's language. If you choose english, you need to use it among the code base.

```suggestion
  pink: '#FF197B',
  backGround: '#F0F0F0',
  blue: '#1F86FF',
```
Also, you can use names as "primary", "Secondary", instead of the colors.